docs/Makefile

```diff
@@ -6,6 +6,7 @@ SPHINXOPTS    =
 SPHINXBUILD   = sphinx-build
 PAPER         =
 BUILDDIR      = ../gh-pages/html
+rtd_BUILDDIR  = _build
```

Comment:
Perhaps try switching this so you have

```
BUILDDIR = _build
GHPAGES_BUILDIR = ....
```
as RTFD probably `export`s the `BUILDDIR` variable